### PR TITLE
add evil-matchit support for ruby files

### PR DESCRIPTION
**Pull request by robbyoconnor**
add evil-matchit support for ruby files

### CODE REVIEW

File: contrib/!lang/ruby/packages.el [hunk 1]
```diff
@@ -55,6 +56,16 @@
                        :post-handlers '(sp-ruby-post-handler (spacemacs/smartparens-pair-newline-and-indent "RET"))
                        :suffix "")))))
 
+(defun ruby/post-init-evil-matchit ()
+  (use-package evil-matchit-ruby
+    :defer t
+    :init
+    (global-evil-matchit-mode 1)
```
Comment (robbyoconnor) [inline contrib/!lang/ruby/packages.el]: Without this line, shit didn't work -- not sure why.

Comment (robbyoconnor) [inline contrib/!lang/ruby/packages.el]: Without this line, it failed to work...

File: contrib/!lang/ruby/packages.el [hunk 2]
```diff
@@ -55,6 +56,17 @@
                        :post-handlers '(sp-ruby-post-handler (spacemacs/smartparens-pair-newline-and-indent "RET"))
                        :suffix "")))))
 
+(defun ruby/post-init-evil-matchit ()
+  (use-package evil-matchit-ruby
+    :defer t
+    :init
+    (progn
+      (global-evil-matchit-mode 1)
```
Comment (robbyoconnor) [inline contrib/!lang/ruby/packages.el]: This was necessary to make it work -- not sure why.

**Comment by robbyoconnor** [comment]
@syl20bnr  ping -- this has been sitting for a long time now :)

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. I had to fix a couple of typos: https://github.com/syl20bnr/spacemacs/commit/4f95d58bd4c2b2e4400251d9dc1cebf37e8977a9.

**Comment by robbyoconnor** [comment]
:+1:

**Pull request closed without merging**